### PR TITLE
HTML & CSS practice: Hooli-style Popup

### PR DESCRIPTION
**Pull request by tavor118sn**
# HTML & CSS practice: Hooli-style Popup

[Demo](https://tavor118sn.github.io/kottans-html-css-popup/) |
[Code base](https://github.com/tavor118sn/kottans-html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/tavor118sn/html-css-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  display: flex;
+}
+
+.popup-menu__more__input:checked + .popup-menu__label {
+  display: none;
+}
+
+.additional-popup-menu {
+  display: none;
+  justify-content: center;
```
Comment (A-Ostrovnyy) [inline submissions/tavor118sn/html-css-popup/style.css:157]: You don't need this line on the hidden element.

File: submissions/tavor118sn/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  clip: rect(0 0 0 0)
+}
+
+/*
+ *
+ * Main navbar
+ *
+ */
+
+header {
```
Comment (A-Ostrovnyy) [inline submissions/tavor118sn/html-css-popup/style.css:41]: Avoid styling by HTML tags. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css).

File: submissions/tavor118sn/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                            Maps
+                        </a>
+                    </li>
+
+                    <!-- More items -->
+                    <li class="popup-menu__more">
+                        <input class="popup-menu__more__input visually-hidden"
+                               type="checkbox"
+                               id="input-popup-menu-more">
+                        <label for="input-popup-menu-more" class="popup-menu__label">
```
Comment (A-Ostrovnyy) [inline submissions/tavor118sn/html-css-popup/index.html:91]: Add styles on focus state.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments above

**Comment by tavor118sn** [comment]
Thanks for comments!
Removed tags, added focus

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**